packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -339,10 +339,9 @@ const getPreviousList = (editor: IEditor, textRange: Range) => {
 const getPreviousListType = (editor: IEditor, textRange: Range, listType: ListType) => {
     const type = listType === ListType.Ordered ? 'orderedStyleType' : 'unorderedStyleType';
     const previousNode = getPreviousList(editor, textRange);
+    const metadata = getMetadata(previousNode.parentElement, ListStyleDefinitionMetadata);
```

Comment:
If previousNode or previousNode.parentElement is null, this will still be a problem.

BTW, what is the parent element here?

Reply:
It should to be the UL or OL element, but I noticed that the list structure can different to just <ol><li></li><ol>, then I changed the code to use findClosestElementAntecesor to find the ul or ol element that contain the metadata.